addons/editor-theme3/addon.json

```diff
@@ -180,14 +184,20 @@
         "addonEnabled": ["debugger"]
       }
     },
+    {
+      "name": "Comments",
+      "id": "comment-color",
+      "type": "color",
+      "default": "#FEF49C"
+    },
     {
       "name": "Block inputs",
       "id": "input-color",
       "type": "color",
       "default": "#FFFFFF"
     },
     {
-      "name": "Text color",
+      "name": "Block text color",
```

Comment:
> This will be much easier to implement when #3805 is merged so maybe we should wait for that.

Yeah, I'll do that. Could you please add the pending label?